src/views/Settings.vue

```diff
@@ -98,19 +98,31 @@
           <ion-card-content>
             {{ $t('The timezone you select is used to ensure automations you schedule are always accurate to the time you select.') }}
           </ion-card-content>
-
-          <ion-item lines="none">
-            <ion-label> {{ userProfile && userProfile.userTimeZone ? userProfile.userTimeZone : '-' }} </ion-label>
-            <ion-button @click="changeTimeZone()" slot="end" fill="outline" color="dark">{{ $t("Change") }}</ion-button>
-          </ion-item>
+            <ion-item lines="none">
+              <ion-label>Browse time zone</ion-label>
```

Comment:
Add translation for static text
```suggestion
              <ion-label>Browser time zone</ion-label>
```